content/practices/personas/index.md

```diff
@@ -76,11 +77,23 @@ why:
    **Tip**: It's helpful to have scissors handy in case you need to cut quadrants apart in order to reassemble
    {{% /callout %}}
 
+10. Plan out how to get access to people that this persona represents and the user research activities that you will do to validate this persona. Check out activities labeled Discovery in the Tanzu Practices center for different options. 
+
+
 ### Success/Expected Outcomes
-Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible.
+Success is when you've created a "master persona" and everyone in the room feels like it represents the target audience as best as possible. Remember that a critical next step is to then validate this persona with [user research](/practices/user-research-introduction-session/) such as interviews
```

Comment:
Let's rephrase "master persona" as something more inclusive, such as "main persona", "primary persona", or similar.

Reply:
let's update the following as well: 
> 9. Tape together the highest scoring quadrants into one "master persona" and reads it to the group